tests/test_storageclass_invalid.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os.path
+
+from resources.ocs import OCS
+from tests import helpers
+from ocs import constants, defaults
+from ocs.exceptions import TimeoutExpiredError
+
+
+logger = logging.getLogger(__name__)
+
+TEMPLATE_DIR = os.path.join('templates', 'ocs-deployment')
+PVC_TEMPLATE = os.path.join(TEMPLATE_DIR, 'PersistentVolumeClaim.yaml')
+
+
+def test_storageclass_cephfs_invalid(invalid_cephfs_storageclass, tmpdir):
+    """
+    Test that Persistent Volume Claim can not be created from misconfigured
+    CephFS Storage Class.
+    """
+    pvc_data = defaults.CSI_PVC_DICT.copy()
+    pvc_name = helpers.create_unique_resource_name('test', 'pvc')
+    pvc_data['metadata']['name'] = pvc_name
+    pvc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
+    pvc_data['spec']['storageClassName'] = invalid_cephfs_storageclass[
+        'metadata']['name']
+    logger.info(
+        f"Create PVC {pvc_name} "
+        f"with storageClassName "
+        f"{invalid_cephfs_storageclass['metadata']['name']}"
+    )
+    pvc = OCS(**pvc_data)
+    pvc.create()
+
+    pvc_status = pvc.get()['status']['phase']
+    logger.debug(f"Status of PVC {pvc_name} after creation: {pvc_status}")
+    assert pvc_status == constants.STATUS_PENDING
+
+    try:
+        logger.info(
+            f"Wait 60 seconds for status of PVC {pvc_name} "
+            f"to change to {constants.STATUS_BOUND} (it shouldn't change)"
+        )
+        pvc_status_changed = pvc.ocp.wait_for_resource(
+            resource_name=pvc_name,
+            condition=constants.STATUS_BOUND,
+            timeout=60,
+            sleep=20
```

Comment:
20 seconds is too much - please change to 3

Reply:
I would leave it as 20 because in this test case it is checked that there is no change after 60 seconds. If sleep is set to 3 then there is log line generated every 3 seconds and logs will become more bloated. Also this way it doesn't call oc command so often and it doesn't consume so much system resources.